ql/termstructures/yield/nonlinearfittingmethods.hpp

```diff
@@ -195,6 +196,50 @@ namespace QuantLib {
         Natural N_;
     };
 
+        //! Natural cubic spline fitting method
+    /*! Fits a discount function using natural cubic spline interpolation
+        where the parameters are nodal discount values d(t_i).
+        The natural boundary condition (second derivative = 0 at ends)
+        is used. If constrainAtZero is true, d(0) is fixed to 1.0 and
+        the parameter vector x contains the remaining nodal values.
+    */
+    class NaturalCubicFitting : public FittedBondDiscountCurve::FittingMethod {
+      public:
+        NaturalCubicFitting(const std::vector<Time>& knotTimes,
+                            bool constrainAtZero = true,
```

Comment:
Do we want to leave the choice?  Without the constraint it doesn't work anyway.  We can remove the argument from the constructors and treat it as always true.